scripts/us_census/geojsons_low_res/generate_mcf.py

```diff
@@ -73,10 +79,12 @@ def generate_simple_geojsons(self, show=False, eps=None):
                     plotter.compare_plots(gpd.read_file(f1),
                                           gpd.read_file(f2),
                                           show=show)
-                    self.simple_geojsons[geoid] = simple
+                    self.simple_geojsons[geoid] = s
             except AssertionError:
                 logging.error("Simplifier failure on GeoJSON below:\n", coords)
-        # plt.show()
+        logging.info(f"Total original points = {original_pts}\n"
+                     f"Compressed points = {compressed_pts}\n")
+        plt.show()
```

Comment:
Do we always want to do this?

Reply:
In the case of logging, I am always doing it since one needs the flag `--alsologtostderr` to display logs anyways. About the `plt.show()` command, this is the command that shows a final plot where one can see all simplified geographies together. Currently, the tool always does this, even if the `show` flag is turned off. Turning the `show` flag on also shows individual comparisons for each simplified geography.